backend/.env.dist.composed

```diff
@@ -1,5 +1,6 @@
 # Kafka settings
 CROWD_KAFKA_BROKERS=kafka:9092
+CROWD_KAFKA_TOPIC=data-sink-worker-normal
```

Comment:
To add context - Crowd was the original name of the system so it's not like CM or CDP. But we have many topics so perhaps maybe name it a bit more not just like `CROWD_KAFKA_TOPIC` but perhaps like `CROWD_KAFKA_DATA_SINK_WORKER_TOPIC` or something like that.